frontend/javascripts/oxalis/view/action-bar/volume_actions_view.js

```diff
@@ -17,9 +17,16 @@ const ButtonGroup = Button.Group;
 
 type Props = {|
   activeTool: VolumeTool,
+  isInMergerMode: boolean,
 |};
 
 class VolumeActionsView extends PureComponent<Props> {
+  componentDidUpdate = (preProps: Props) => {
```

Comment:
```suggestion
  componentDidUpdate = (prevProps: Props) => {
```

It's usually called `prevProps` - meaning previous props :)